.idea/copyright/Enterprise.xml

```diff
@@ -0,0 +1,6 @@
+<component name="CopyrightManager">
```

Comment:
Name file `Element_Enterprise` so it's aligned?

Reply:
Yes good idea, but not a big deal. I'll rename it later.